homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -59,7 +60,7 @@ def __init__(self, api, name):
         """Initialize the MeteoAlert binary sensor."""
         self._name = name
         self._attributes = {}
-        self._state = None
```

Comment:
Revert that. The state should only be set by `update()`.

Reply:
Is it a cardinal sin to initialize member variables for their intended purpose in Python?
Old-school C++ dev, here ;)